server/rscompat.cpp

```diff
@@ -1075,6 +1075,24 @@ void rscompat_postprocess(struct rscompat_info *info)
   autoadjust_ruleset_data();
 }
 
+/**
+ * Adds <VisionLayer, Main, Local, True> req to all unit vision reqs,
+ * as compat for missing CAP_VUT_VISIONLAYER
+ */
+static bool rscompat_vision_effect_cb(struct effect *peffect, void *data)
+{
+  struct rscompat_info *info __attribute__((unused)) =
```

Comment:
`__attribute__` is a GNU extension. Use `Q_UNUSED(variable_name);` or the standard `[[maybe_unused]]` instead.